config/local/config.yml

```diff
@@ -53,6 +53,8 @@ chains:
     quick_start_num_blocks_back: 300000
     num_block_confirmations_before_fill: <num_block_confirmations_before_fill> # e.g. 1
     max_rebalancing_gas_cost_uusdc: "1000000"
+    fund_rebalancing_timeout: 1h
+    fund_rebalancing_cost_cap_uusdc: "2000000"
```

Comment:
i feel like these should be moved to the fund rebalancer config and then drop the `fund_rebalancer_` prefix